apollo-router/tests/snapshots/apollo_reports__send_variable_value.snap

```diff
@@ -193,363 +193,355 @@ traces_per_query:
                         seconds: "[seconds]"
                         nanos: "[nanos]"
                 - node:
-                    Condition:
-                      condition: sendValue
-                      if_clause:
+                    Flatten:
```

Comment:
@Geal I've tagged you as reviewer since this needs attention and i /think/ you worked on it for the auth pr